packages/zosmf/src/constants/Zosmf.constants.ts

```diff
@@ -67,6 +67,15 @@ export const ZosmfConstants: { [key: string]: any } = {
          * @type {string}
          */
         UNABLE_TO_VERIFY_LEAF_SIGNATURE: "UNABLE_TO_VERIFY_LEAF_SIGNATURE"
+    },
+
+    VERSIONS: {
```

Comment:
I think it'd be ideal to add a link here:
```suggestion
    // https://www.ibm.com/docs/en/zos/2.5.0?topic=service-retrieve-zosmf-information
    VERSIONS: {
```